### PR TITLE
Edit puml and images

### PR DESCRIPTION
**Pull request by amoonguss1**
This PR will update the images file with the correct notations for the DG. 
Updates the sequence diagram for CompanyCommand.
Partly resolves issue #159 and #260

### CODE REVIEW

File: docs/images/ArchitectureSequenceDiagram.png
Comment (dsja612) [inline docs/images/ArchitectureSequenceDiagram.png:1]: Good change. Lets check if the reduced image dimension might make the diagram hard to view in the DG.

File: docs/images/ModelClassDiagram.png
Comment (dsja612) [inline docs/images/ModelClassDiagram.png:1]: Should be `company` instead of `address`

**Review by dsja612 — CHANGES_REQUESTED** [review]
BetterModelClassDiagram needs a small fix and LGTM

File: docs/images/ModelClassDiagram.png
Comment (amoonguss1) [reply docs/images/ModelClassDiagram.png:1]: fixed as of the latest commit. Thanks for pointing it out!

**Review by amoonguss1 — COMMENTED** [review]
LGTM!

**Review by dsja612 — APPROVED** [review]

File: docs/diagrams/BetterModelClassDiagram.puml
Comment (RyanQiu1) [inline docs/diagrams/BetterModelClassDiagram.puml:1]: LGTM, changing phone to contact makes it more suited.

**Pull request merged**